packages/extension-chakra-storefront/src/components/display-price/current-price.jsx

```diff
@@ -33,7 +33,7 @@ const CurrentPrice = ({labelForA11y, price, as, isRange = false, currency, ...ex
                 <Text
                     as={as}
                     {...extraProps}
-                    aria-live={'polite'}
```

Comment:
FYI: String Literal Format is preferred when the value is a static string. And less overhead since it doesn't need to evaluate JavaScript expressions